integrationtests/cli/apply/apply_test.go

```diff
@@ -179,4 +179,38 @@ var _ = Describe("Fleet apply", Ordered, func() {
 			}).Should(BeTrue())
 		})
 	})
+
+	When("non-helm type bundle uses helm options in fleet.yaml", func() {
+		When("passes along enabled helm options", func() {
+			BeforeEach(func() {
+				name = "helm_options_enabled"
+				options = apply.Options{Output: gbytes.NewBuffer()}
+				dirs = []string{cli.AssetsPath + name}
+			})
+
+			It("publishes the flag in the bundle options", func() {
+				Eventually(func() bool {
+					bundle, err := cli.GetBundleFromOutput(options.Output)
+					Expect(err).NotTo(HaveOccurred())
+					return bundle.Spec.Helm.TakeOwnership && bundle.Spec.Helm.Atomic && bundle.Spec.Helm.Force && bundle.Spec.Helm.WaitForJobs && bundle.Spec.Helm.DisablePreProcess && bundle.Spec.Helm.ReleaseName == "enabled"
```

Comment:
This looks somewhat easier to ready to me (as would be future diffs on this):
```suggestion
					return bundle.Spec.Helm.TakeOwnership &&
					        bundle.Spec.Helm.Atomic &&
					        bundle.Spec.Helm.Force &&
					        bundle.Spec.Helm.WaitForJobs &&
					        bundle.Spec.Helm.DisablePreProcess &&
					        bundle.Spec.Helm.ReleaseName == "enabled"
```

Same remark about L211.